### PR TITLE
Add project type filter to outflow report

### PR DESCRIPTION
**Pull request by gigxz**

### CODE REVIEW

File: app/views/warehouse_reports/outflow/_filter.haml [hunk 1]
```diff
@@ -7,31 +7,36 @@
         = f.input :end, label: 'End Date', as: :date_picker, input_html: {value: @filter.end}
```
Comment (gigxz) [inline app/views/warehouse_reports/outflow/_filter.haml:7]: I reorganized the filters a bunch to make it cleaner (hopefully). Does this organization and labeling make sense @eanders ? Also might be jarring for folks if they're used to filters being in certain places. Happy to move it back if you think.

Screen Shot 2022-12-09 at 11 24 31 AM

Comment (gigxz) [reply app/views/warehouse_reports/outflow/_filter.haml:7]: Also I changed `No Project Limit` -> `No Additional Projects` so it matches the label.  But looking at the code, it looks like it actually is a limit, not "additional projects" being added..
https://github.com/greenriver/hmis-warehouse/blob/8e81d7a48a0b708dfe5382b43793511bc5ee33d1/app/models/grda_warehouse/warehouse_reports/outflow_report.rb#L227-L247

Should it be "Projects _excluded_ from the "without recent service" calculation.?

Comment (eanders) [reply app/views/warehouse_reports/outflow/_filter.haml:7]: That's "make sure clients don't have service in these projects" right?
The re-work of the filters looks great.  Thanks!

Comment (gigxz) [reply app/views/warehouse_reports/outflow/_filter.haml:7]: Yeah, I guess that's what it's doing. Should I change the wording, or does that placeholder text make sense?

Comment (eanders) [reply app/views/warehouse_reports/outflow/_filter.haml:7]: Let's label it:
"Additional Projects used to exclude clients when checking "without recent service"
and the placeholder can stay as you have it.
Does that read ok to you?

Comment (gigxz) [reply app/views/warehouse_reports/outflow/_filter.haml:7]: Yeah, I think that's better. Updated. thanks

**Pull request merged**